.github/workflows/release.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
-          platforms: ${{ github.event.ref_type == 'tag' && 'linux/amd64,linux/arm,linux/arm64' || 'linux/amd64' }}
```

Comment:
I think linux/arm is needed for some armbian distros, isn't it? I know rasp pi 5 is arm64, but many sbc's are not

Reply:
I noticed that the cloudflared container is built only for amd64 and arm64 architectures. Therefore, I suggest we build the operator for these architectures as well.

Reply:
makes sense to me